package.json

```
@@ -63,7 +63,8 @@
 			"jsdoc/require-returns": "off",
 			"jsdoc/require-param-description": "off",
 			"jsdoc/no-undefined-types": "off",
-			"jsdoc/require-returns-description": "off"
+			"jsdoc/require-returns-description": "off",
+      "no-unused-vars": "off"
```

Watch your spaces/tabs here.

```suggestion
			"no-unused-vars": "off"
```

---

Thanks -- fixed!